swim/node.go

```diff
@@ -510,3 +510,7 @@ func (n *Node) pingNextMember() {
 		"numErrors": len(errs),
 	}).Warn("ping request inconclusive due to errors")
 }
+
```

Comment:
I would be fine with PingableMembers as well, I'd prefer to keep the GetMembers on the Ringpop struct though since the application embeding Ringpop does not really care about the pingable concept in my opinion, they just want a list of members from ringpop, they only care about members that are currently in working state.